src/runtime/pkg/containerd-shim-v2/create.go

```diff
@@ -223,6 +230,10 @@ func create(ctx context.Context, s *service, r *taskAPI.CreateTaskRequest) (*con
 			}
 		}()
 
+		// CDI annotations have been processed during PodSandbox creation and cold-plug.
+		// CDI annotations referencing device kinds that exist in the guest (e.g., nvidia.com/gpu) will be generated during device attachment.
```

Comment:
Long lines of ocmments are hard to read break them down to ~80 characters lenght (there is a vscode ext for that -- Alt + q)